.github/workflows/reuse-run-e2e-tests.yml

```diff
@@ -133,7 +133,7 @@ jobs:
           docker container ls -a
       - name: Run e2e tests
         id: run_e2e_tests
-        timeout-minutes: 4
+        timeout-minutes: 6
```

Comment:
The liveness e2e needs to be started after the usual e2e tests ended because the sequencer will be restarted in liveness e2e which might disrupt the result of usual e2e tests if they are both running in parallel. Usually the liveness test alone only takes 30 sec but added timeout to 6 mins just to avoid flaky liveness tests as we'd seen sequencer took a bit longer time to return `eth_getLogs` result after restart